src/config.ts

```diff
@@ -53,66 +47,19 @@ const getEuSigner = () => ({
     process.env.SIGNING_EU_QR_NAME,
     sampleSigningDidName
   ),
-  publicKey: getDefaultIfUndefined(
-    process.env.SIGNING_EU_QR_PUBLIC_KEY?.replace(/\\n/g, "\n"),
-    ""
+});
+
+const getGPayCovidCardSigner = async () => ({
```

Comment:
Could you help me remove the async/await since it's no longer necessary? Thanks!

Reply:
We could also call the `getGPayCovidCardSigner`**()** directly already:
```javascript
gpaySigner: getGPayCovidCardSigner(),
```
https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/5b7742a0685a039ac50d142cb31bcb7aa90490f6/src/config.ts#L123